hnn_core/tests/test_compare_hnn.py

```diff
@@ -21,15 +30,39 @@ def test_hnn_core():
 
     hnn_core_root = op.join(op.dirname(hnn_core.__file__), '..')
 
+    # default params
     params_fname = op.join(hnn_core_root, 'param', 'default.json')
     params = Params(params_fname)
 
-    net = Network(params, n_jobs=1)
-    dpl = simulate_dipole(net)[0]
+    # run the simulation
+    dpl = run_simulation(params, dpl_master)[0]
 
+    # write the dipole to a file and compare
     fname = './dpl2.txt'
     dpl.write(fname)
 
     dpl_pr = loadtxt(fname)
     assert_array_equal(dpl_pr[:, 2], dpl_master[:, 2])  # L2
     assert_array_equal(dpl_pr[:, 3], dpl_master[:, 3])  # L5
+
+
+if __name__ == '__main__':
+    # started as an MPI child from test_mpi_simulation.py
+
+    from mpi4py import MPI
```

Comment:
you will also need to update the contributing guide to install this dependency

Reply:
Thanks!

Reply:
Done